source/local/local.go

```diff
@@ -272,8 +297,20 @@ func getFeaturesFromFiles() (map[string]string, error) {
 			continue
 		}
 
+		// Parse comments
+		comments := parseComments(lines, fileName)
+
+		// handle expiry date
+		if expiryDate, ok := comments[ExpiryDateKey]; ok {
+			if expiryDate.(time.Time).Before(time.Now()) {
```

Comment:
We need to take `time.Now()` in the beginning of this function so that (for consistency) the same timestamp is used for all lines in the file